torch/_inductor/codegen/triton.py

```diff
@@ -1546,7 +1546,7 @@ def _mask_value(value, default):
                 sum_ = ops.reduction(dtype, dtype, "sum", value)
                 self.inside_reduction = False
                 rnumel = ops.index_expr(self.numels[-1], dtype)
-                mean = ops.div(sum_, rnumel)
+                mean = ops.truediv(sum_, rnumel)
```

Comment:
I'm confused where truediv is defined for triton? The tests are clearly passing but I can't find it anywhere.

Reply:
Oh I think it fallls all the way back to `MockHandler` in which case it definitely does `a / b` which in triton is C division not truediv.

Reply:
Actually I take it back, it is true div in triton. It's floor div that triton handles incorrectly.